dist/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script><script defer src="print.bundle.js"></script></head>
+<body>
+    <h1>Leaderboard</h1>
+</body>
+</html>
```

Comment:
- [ ] Kindly make sure you write the  plain HTML markup with minimum styling (just to make the layout). You should use the provided  wireframe as a guide 

![Screenshot from 2023-02-01 02-10-05](https://user-images.githubusercontent.com/80458199/215913404-57bbe9a5-f788-48f7-9459-5bd14d99a4de.png)


![# STATUS: REQUIRED CHANGES :recycle::recycle: ](https://user-images.githubusercontent.com/80458199/215913412-c1209394-ada7-4bf2-a329-7648689b4437.png)

Reply:
hey mr@ iambenkis
this is what am seeing
![Screenshot (257)](https://user-images.githubusercontent.com/70164878/215915230-fb3a898e-f14f-4864-97ea-cc94fa40fe6a.png)